content/tools/data.js

```diff
@@ -43,6 +43,24 @@ const tools = [
       "California's 4th Climate Change Assessment",
     ],
   },
+  {
+    title: "Comparative Sea Level Rise",
```

Comment:
Update the title. At the last TAC meeting, Juliette mentioned not to use the word Comparative because we don’t want users to compare models. The preference was for the title to reference SLR & Extreme Flooding. Although the word Extreme also has some interpretation issues according to other TAC members. One potential option for title could be "Coastal Flooding Scenarios from Sea Level Rise"

Reply:
Thanks, I guess we should also update the slug then as well.